src/doblib/__main__.py

```diff
@@ -136,6 +150,12 @@ def main(args=None):
     elif args.command in ("a", "action"):
         # Run actions in the environment
         sys.exit(ActionEnvironment(args.cfg).apply_action(left))
+    elif args.command in ("m", "migrate"):
+        # Run Odoo migration using OpenUpgrade
+        migrate_args, left = load_migrate_arguments(left)
+        version = migrate_args.version
+        migrate_cfg = f"odoo.migrate.{version}.yaml"
+        sys.exit(MigrateEnvironment(migrate_cfg).migrate(version, left))
```

Comment:
Just pass the `migrate_args` instead of `version` and `left`. If we later need flags we can skip the argument list.